.github/workflows/release-suite-config.yml

```diff
@@ -0,0 +1,48 @@
+name: "[Release] suite config"
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
```

Comment:
I don't understand where the environment value comes from, but the production config release should be allowed on codesign branch only.

Reply:
evnironment values is defiend in github repository settings and its what allows us to use more granular permissions and require one or multiple approvals even for jobs. So anyone can trigger this but only defined environment approved would be able to make this action run and complete what is described. We wont use codesing branch anymore id say. workflow_dispatch uses main branch as the base or we could also add if condition to the job so it only works if its triggered for a certain branch only.

Reply:
thanks for explanation 👍